etc/env/base/house_keeper.toml

```diff
@@ -13,4 +13,6 @@ fri_prover_stats_reporting_interval_ms = 30000
 fri_proof_compressor_job_retrying_interval_ms = 30000
 fri_proof_compressor_stats_reporting_interval_ms = 10000
 fri_prover_job_archiver_reporting_interval_ms = 1800000
-fri_prover_job_archiver_archiving_interval_ms = 172800
\ No newline at end of file
+fri_prover_job_archiver_archiving_interval_ms = 172800
+fri_gpu_prover_archiver_reporting_interval_secs = 86400
+fri_gpu_prover_archiver_archiving_interval_secs = 172800
```

Comment:
Hmm, I don't get why it can bring provers down. Could you elaborate?

Reply:
But provers cannot just "resurrect" themselves. If they are dead - they are dead anyway. If they will be killed - new will be spawned. Or am I missing something?

Reply:
But in that case we need to keep possible None status. Right, that just came to my mind! That's why we would need to check not only that status is dead, but also that it exists in the table. Since availability checker and archiver are not bound with each other - at some point archiver can archive the prover - after that the check will occur and it will panic. I don't think that it really depends on the value that we set - it's just possible.

Reply:
side note: In additional to the potential comment I would really revisit the variable naming - see above. Currently it indeed reads like an interval on which something's happening, and it does feel innocent to reduce it. If we call it something like `max_age`, it's much more clear that this might have business logic reprecussions